coalib/bearlib/__init__.py

```diff
@@ -60,7 +60,7 @@ def deprecate_settings(**depr_args):
      ['old']
 
```

Comment:
Also, the commit body should be "Changes" instead of "Changed".

the words nondeterministic and the new one should be each enclosed in double backticks: `` (these are double backticks) 

and the commit body should end with a period.